src/main/java/com/fryrank/dal/ReviewDALImpl.java

```diff
@@ -50,6 +51,16 @@ public GetAllReviewsOutput getAllReviewsByAccountId(@NonNull final String accoun
         return new GetAllReviewsOutput(reviews);
     }
 
+    @Override
+    public GetAllReviewsOutput getTop10MostRecentReviews(){
+        final Query query= new Query();
+        query.with(Sort.by(Sort.Direction.DESC, "isoDateTime"));
```

Comment:
This `isoDateTime` string is also used in other places; can we put it in a constants file?

Reply:
placed in constants file